modules/installation-special-config-storage.adoc

```diff
@@ -127,44 +170,100 @@ boot_device:
+. Create a control plane or compute node manifest from the corresponding Butane config and save it to the `<installation_directory>/openshift` directory. For example, to create a manifest for the compute nodes, run the following command:
 +
 [source,terminal]
 ----
-$ butane $HOME/clusterconfig/worker-storage.bu -o ./99-worker-storage.yaml
+$ butane $HOME/clusterconfig/worker-storage.bu -o <installation_directory>/openshift/99-worker-storage.yaml
 ----
++
+Repeat this step for each node type that requires disk encryption or mirroring.
 
-. Save the Butane config in case you need to update the manifest in the future.
-
-. Continue with the remainder of the {product-title} deployment.
+. Save the Butane configs in case you need to update the manifests in the future.
 
+. Continue with the remainder of the {product-title} installation.
++
 [IMPORTANT]
 ====
 If you configure additional data partitions, they will not be encrypted unless encryption is explicitly requested.
 ====
+
+.Verification
+
+After installing {product-title}, you can verify if the boot disk encryption or mirroring is enabled on each cluster node.
+
+. From the installation node as a user that has SSH access to your cluster, review the status of the root mapping on a cluster node:
++
+[source,terminal]
+----
+$ ssh core@compute-1.example.com sudo cryptsetup status root
+----
++
+.Example output
+[source,terminal]
+----
+/dev/mapper/root is active and is in use.
+  type:    LUKS2 <1>
+  cipher:  aes-xts-plain64 <2>
+  keysize: 512 bits <3>
+  key location: keyring
+  device:  /dev/sda4 <4>
+  sector size:  512
+  offset:  32768 sectors
+  size:    15683456 sectors
+  mode:    read/write
+----
+<1> The encryption format. When the TPM v2 or Tang encryption modes are enabled, the {op-system} boot disks are encrypted using the LUKS2 format.
+<2> The encryption algorithm used to encrypt the LUKS2 volume. The `aes-cbc-essiv:sha256` cipher is used if FIPS mode is enabled.
+<3> The cryptographic key size in bits.
+<4> The device that contains the encrypted LUKS2 volume. If mirroring is enabled, the value will represent a software mirror device, for example `/dev/md126`.
+
+. If you have enabled boot disk encryption, list the Clevis plug-ins that are bound to the encrypted device:
++
+[source,terminal]
+----
+$ ssh core@compute-1.example.com sudo clevis luks list -d /dev/sda4 <1>
+----
+<1> Specify the device that is listed in the `device:` field in the output of the preceding step.
++
+.Example output
+[source,terminal]
+----
+1: sss '{"t":1,"pins":{"tang":[{"url":"http://tangserver.example.com:7500"}]}}' <1>
+----
+<1> In the example output, the Tang plug-in is used by the Shamir's Secret Sharing (SSS) Clevis plug-in for the `/dev/sda4` device.
+
+. Repeat the validation steps for each {op-system} node in your cluster.
```

Comment:
For each node type.  It shouldn't be necessary to perform the validation on each node.

Also, I believe the MCO will degrade a node if it's accessed over SSH.

Reply:
@bgilbert thank you for your detailed review!

I will update this line so that it suggests validation on one of each node type. Also, I will change the validation steps so that they use the `oc debug node` facility instead of SSH.

Reply:
I have now changed the verification procedure to use a debug shell instead of SSH. I have also updated the last step in the procedure as per your recommendation. Thanks!